src/client/backends/Msfs/Mapping.ts

```diff
@@ -158,7 +162,7 @@ export class MsfsMapping {
                     magneticBearing: this.trueToMagnetic(bearing, magVar),
                     gradient: primary ? gradient : -gradient,
                     startLocation,
-                    thresholdLocation,
+                    location: thresholdLocation,
```

Comment:
This doesn't need to be a fix. This is data about a runway assocaited with an airport... not a fix associated with a procedure... that should be something different (e.g. that's what mapRunwayFix is for creating). Renaming this particular field loses linkage with associated fields (i.e. thresholdCrossingHeight).